symphco/engineering-standards
# docs: Add Secrets Manager

## What does this PR do?
- Migrate secrets manager docs from notion to GitHub

## Why did you choose this solution?
- GitHub provides a more structured and secure environment for managing secrets

## What is the dependency graph of the changes?
- Changes are primarily to the file and asset structure within the project repository, involving relocations and renames of secret-related assets

## How to all test/verify?
- Ensure images and markdown links redirect appropriately within GitHub. Check if environment variables within Cloud Run integrate without issues.

## Related ticket url:
- None provided
